### DISCUSSION TITLE
Can I publish the guide to install Shairport if I credit?

### DISCUSSION THREAD

**Discussion by outsharded** [mikebrady/shairport-sync#1318] [Q&A] [answered]
@mikebrady, I wanted to ask wether I can publish the guides to install Shairport-sync on [my website](https://tec-kids.co.uk), providing that I credit you?

Thanks

**Comment by mikebrady** [comment, marked as answer]
Sure thing, and the credit would be appreciated, thanks.

One little thing that tends to happen with the passage of time is that guides sometimes go out of date, so let me suggest a line of text, something like, "For the latest details, please visit the [Shairport Sync repository](https://github.com/mikebrady/shairport-sync)." or something like that.

Thanks again.

**Reply by outsharded** [reply]
Thanks!
